src/Connection/ServiceConnection.ts

```diff
@@ -191,6 +198,26 @@ export class ServiceConnection {
                 ConnectionManager.messageReceiver.lastInteractionZoneUpdate = { status: 'UNPROCESSED', state: state };
                 break;
             }
+
+            case ActionCode.LICENSE_STATE: {
```

Comment:
💬 When is LICENSE_STATE sent vs LICENSE_STATE_RESPONSE? Is the first on initialisation and the second when a request to get the license state is sent?

Reply:
`LICENSE_STATE_RESPONSE` is the response to `GET_LICENSE_STATE`, `LICENSE_STATE` is the event of "hey license changed - here it is". I think currently you'll get _both_ after requesting license state because I'm sending the event on set, not on _change_, but I can fix that quickly if we think it's worth doing?

Reply:
I thought we only get license changes when requested so still unsure when only LICENSE_STATE  is sent (and LICENSE_STATE_RESPONSE isn't). 

If we are always sending both then makes sense to combine into one response?

Reply:
Other ways License State could change:

* If another TouchFree client requests a change
* If another Tracking client requests a change
* If you place a License file in directly, as you have to for Business Licensing

Also, if you `GetLicenseState` you will get both, but if the License State changes, you won't get the `RESPONSE` because that wouldn't be a response _to_ anything. To phrase this another way; LICENSE_STATE would stay on the Websocket as it's notifying info from Service, `LICENSE_STATE_RESPONSE` could be replaced with a HTTP response.

Finally, you don't get a `LicenseState` in a `ADD_LICENSE_RESPONSE` or `REMOVE_LICENSE_RESPONSE`, those are the cases where the "oh licensing changed" event becomes useful (this is because there's several layers of async between tooling and the tracking diagnostic API, and then the licensing service)